spec/services/smoke_test/test_emails_spec.rb

```diff
@@ -24,5 +24,11 @@
       subject
       ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper.drain
     end
+
+    it 'sends a notify_provider email' do
+      expect_any_instance_of(SubmitApplicationReminderMailer).to receive(:notify_provider).and_call_original
```

Comment:
Why `.add_call_original`? The expectation is fulfilled on calling the method, so what happens afterwards is not needed for the expectation to have been satisfied. And what you want to avoid is the system accidentally sending out emails. So I'd suggest doing the simpler `.and_return(true)`